test/src/test/java/org/corfudb/runtime/view/stream/StreamAddressSpaceTest.java

```diff
@@ -217,7 +217,7 @@ public void testToArray() {
     @Test
     @SuppressWarnings("ConstantConditions")
     public void testEquality() {
-        assertThat(new StreamAddressSpace().equals(null)).isFalse();
+        assertThat(new StreamAddressSpace() == null).isFalse();
```

Comment:
This was explicitly testing the overridden equals method. If I make it return `true` instead of `false`, the test now passes when it shouldn't.

Reply:
My bad, will revert it.